packages/babel-preset-env/src/utils.js

```diff
@@ -62,7 +62,7 @@ export function findSuggestion(options: string[], option: string): string {
       return validOption;
     }
     return suggestion;
-  }, undefined);
+  }, "");
```

Comment:
`Cannot return options.reduce(...) because undefined [1] is incompatible with string [2].`